jinad/helper.py

```diff
@@ -0,0 +1,65 @@
+import copy
+from typing import Union, Tuple, List
+from jina.flow import Flow as _Flow
+from jina import __default_host__
+from jina.peapods.pod import FlowPod as _FlowPod
+from jina.peapods.pea import BasePea
+from jina.excepts import FlowTopologyError
+
+
+class Flow(_Flow):
```

Comment:
Shouldnt' it be better to have a `has a Flow` dependency rather than `it is a Flow` logic?

Reply:
Why isn't it just a `static` method getting a `flow` as argument instead of `self`?

Reply:
This will be refactored

Reply:
`Flow.add()` invokes `FlowPod`. `FlowPod` calls `RemoteMutablePod`. Here we have implemented `RemoteMutablePod`. Hence this is to import the right class. This will be refactored with right inheritance so that we don't have repeated code.